probe-rs-tools/src/bin/probe-rs/util/cli.rs

```diff
@@ -280,15 +280,12 @@ pub async fn monitor(
         print_monitor_event(&mut rtt_client.as_mut(), msg)
     });
 
-    let mut cancelled = false;
-
     let result = with_ctrl_c(monitor, async {
-        cancelled = true;
         session.client().publish::<CancelTopic>(&()).await.unwrap();
     })
     .await;
 
-    if cancelled && print_stack_trace {
```

Comment:
Does this not also print the stacktrace on regular exit?

Reply:
No, `cancelled` is only set by ctrl-c.